src/to-be-disabled.js

```diff
@@ -41,6 +41,10 @@ function isElementDisabled(element) {
   return FORM_TAGS.includes(getTag(element)) && element.hasAttribute('disabled')
 }
 
+function hasAriaDisabled(element) {
```

Comment:
nit pick: I'd rather call this `isAriaDisabled`.

Reply:
updated :)